### PR TITLE
Translated writing data - r-novice-gapminder

### PR DESCRIPTION
**Pull request by atusy**

### CODE REVIEW

File: po/r-novice-gapminder.ja.po [hunk 1]
```diff
@@ -15630,31 +15630,31 @@ msgstr ""
...
-"keypoints:\n"
-"- \"Save plots from RStudio using the 'Export' button.\"\n"
-"- \"Use `write.table` to save tabular data.\"\n"
+"title: データの出力\n"
+"講義: 10\n"
+"練習: 10\n"
+"質問:\n"
+"- \"Rで作った図を保存するにはどうしますか？\"\n"
+"目標:\n"
+"- \"図やデータを R から書き出せるようになりましょう。\"\n"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]: I am not sure which form ("しよう” or "しましょう”). Either is fine by me. Just for the sentence, my suggestion is:

```suggestion
"- \"R から図やデータを 出力出来るようになりましょう。\"\n"
```

File: po/r-novice-gapminder.ja.po [hunk 2]
```diff
@@ -15673,9 +15673,8 @@ msgid ""
 "in the 'Plot' window. This will give you the option of saving as a\n"
 ".pdf or as .png, .jpg or other image formats."
 msgstr ""
-"You can save a plot from within RStudio using the 'Export' button\n"
-"in the 'Plot' window. This will give you the option of saving as a\n"
-".pdf or as .png, .jpg or other image formats."
+"RStudio から図を保存するには \"Plot\" ウィンドウにある \"Export\" ボタンを使います。\n"
+"この機能を使うと、図を .pdf や .png、.jpg など様々が画像形式を選ぶことができます。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po:15525]:
```suggestion
"この機能を使うと、図を .pdf や .png、.jpg など様々な画像形式で保存することができます。"
```

File: po/r-novice-gapminder.ja.po [hunk 3]
```diff
@@ -15686,22 +15685,20 @@ msgid ""
...
-"Sometimes you will want to save plots without creating them in the\n"
-"'Plot' window first. Perhaps you want to make a pdf document with\n"
-"multiple pages: each one a different plot, for example. Or perhaps\n"
-"you're looping through multiple subsets of a file, plotting data from\n"
-"each subset, and you want to save each plot, but obviously can't stop\n"
-"the loop to click 'Export' for each one."
+"図を \"Plot\" ウィンドウに表示することなく保存したいこともあるでしょう。\n"
+"ページごとに一つの図を含むような複数のページから成る PDF を出力したいこともあるでしょう。\n"
+"あるいはループを使ってファイル中のデータの絞り込み条件を変えながら複数の図を作成していて、\n"
+"一つずつ保存するために一々ループをとめて \"Export\" ボタンを押すことが不可能な場合もあるでしょう。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]: As I did not know how to read :)
```suggestion
"一つずつ保存するためにいちいちループを止めて \"Export\" ボタンを押すことが不可能な場合もあるでしょう。"
```

File: po/r-novice-gapminder.ja.po [hunk 4]
```diff
@@ -15723,14 +15720,14 @@ msgstr ""
...
-"# You then have to make sure to turn off the pdf device!\n"
+"# PDF デバイスを確実に終了しておく必要があります！\n"
 "\n"
 "dev.off()\n"
 "~~~"
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:59
 msgid "Open up this document and have a look."
-msgstr "Open up this document and have a look."
+msgstr "このドキュメントを開いて内容を確認して見て下さい。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
msgstr "このドキュメントを開いて内容を確認してみましょう。"
```

File: po/r-novice-gapminder.ja.po [hunk 5]
```diff
@@ -15753,10 +15750,8 @@ msgid ""
 "> > {: .language-r}"
 msgstr ""
 ">\n"
-"> Rewrite your 'pdf' command to print a second\n"
-"> page in the pdf, showing a facet plot (hint: use `facet_grid`)\n"
-"> of the same data with one panel per continent.\n"
-"> > ## Solution to challenge 1\n"
+"> `pdf` コマンドを書き直して、2ページ目に同じデータを大陸ごとにファセットした図を出力してみましょう (ヒント:  `facet_grid` を使います)。\n"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"> `pdf` コマンドを書き直して、2ページ目に同じデータを `continent` ごとにファセットした図を出力してみましょう (ヒント:  `facet_grid` を使います)。\n"
```

File: po/r-novice-gapminder.ja.po [hunk 6]
```diff
@@ -15774,34 +15769,30 @@ msgstr ""
...
-msgstr ""
-"The commands `jpeg`, `png` etc. are used similarly to produce\n"
-"documents in different formats."
+msgstr "同様にして `jepg` や `png` などのコマンドを使うことで、他の形式のドキュメントを作成できます。"
 
 # header
 #: r-novice-gapminder/_episodes/11-writing-data.md:86
 msgid "## Writing data"
-msgstr "## Writing data"
+msgstr "## データを書き出す"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po:15625]: Just to follow the title:
```suggestion
msgstr "## データの出力"
```

File: po/r-novice-gapminder.ja.po [hunk 7]
```diff
@@ -15774,34 +15769,30 @@ msgstr ""
...
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:88
 msgid "At some point, you'll also want to write out data from R."
-msgstr "At some point, you'll also want to write out data from R."
+msgstr "R からデータを書き出したいこともあります。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po:15629]:
```suggestion
msgstr "R からデータを出力したいこともあります。"
```

File: po/r-novice-gapminder.ja.po [hunk 8]
```diff
@@ -15774,34 +15769,30 @@ msgstr ""
...
+msgstr "これには、既に使った `read.table` 関数とよく似た `write.table` 関数を使います。"
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:93
 msgid ""
 "Let's create a data-cleaning script, for this analysis, we\n"
 "only want to focus on the gapminder data for Australia:"
 msgstr ""
-"Let's create a data-cleaning script, for this analysis, we\n"
-"only want to focus on the gapminder data for Australia:"
+"データを綺麗にするスクリプトを書きましょう。\n"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po:15642]: Just to make sure if 綺麗 is commonly used for "data-cleaning" in Japanese, 
```suggestion
"データを綺麗にするスクリプトを書きましょう。\n"
```

File: po/r-novice-gapminder.ja.po [hunk 9]
```diff
@@ -15879,17 +15868,14 @@ msgid ""
 "quotation marks come from? Also the row numbers are\n"
 "meaningless."
 msgstr ""
-"Hmm, that's not quite what we wanted. Where did all these\n"
-"quotation marks come from? Also the row numbers are\n"
-"meaningless."
+"どうやら望んだ通りに出力されていません。引用符はどこから来たのでしょうか。\n"
+"また、無意味な行番号があります。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"また、行番号には意味がありません。"
```

File: po/r-novice-gapminder.ja.po [hunk 10]
```diff
@@ -15879,17 +15868,14 @@ msgid ""
...
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:137
 msgid ""
 "Let's look at the help file to work out how to change this\n"
 "behaviour."
-msgstr ""
-"Let's look at the help file to work out how to change this\n"
-"behaviour."
+msgstr "ヘルプファイルを見てこの挙動の変更を試みましょう。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
msgstr "ヘルプファイルを見てこの動作を変更しましょう。"
```

File: po/r-novice-gapminder.ja.po [hunk 11]
```diff
@@ -15937,7 +15922,7 @@ msgstr ""
 
 #: r-novice-gapminder/_episodes/11-writing-data.md:162
 msgid "Now lets look at the data again using our shell skills:"
-msgstr "Now lets look at the data again using our shell skills:"
+msgstr "もう一度シェルの技術を使ってデータを確認してみましょう。"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
msgstr "もう一度シェルを使ってデータを確認してみましょう。"
```

File: po/r-novice-gapminder.ja.po [hunk 12]
```diff
@@ -15993,12 +15978,10 @@ msgid ""
...
 msgstr ""
 ">\n"
-"> Write a data-cleaning script file that subsets the gapminder\n"
-"> data to include only data points collected since 1990.\n"
+"> gapminder データを1990年以降のデータのみに絞り込むスクリプトを書いて下さい。\n"
 ">\n"
-"> Use this script to write out the new subset to a file\n"
-"> in the `cleaned-data/` directory.\n"
-"> > ## Solution to challenge 2\n"
+"> このスクリプトを使って絞り込んだ結果を `cleaned-data/` ディレクトリ中のファイルに書き出しましょう。\n"
```
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"> このスクリプトを使って絞り込んだ結果を `cleaned-data/` ディレクトリ中のファイルに出力しましょう。\n"
```

**Review by masamiy — COMMENTED** [review]
Hi!
Excellent work!! Thank you so very much!
I just made a few suggestions. None of them are critical.

File: po/r-novice-gapminder.ja.po (hunk 8 above)
Comment (atusy) [reply po/r-novice-gapminder.ja.po:15642]: Thanks. I will consider a bit for the better translation

**Comment by TomKellyGenetics** [comment]
Thanks @atusy. Sorry it's taken so long for us to get on top of the backlog of lessons to review. We are close to a full version of the R lessons. 

We are still looking for translators for the last 3 lessons on the tidyverse if you are able to do another one?
https://github.com/swcarpentry-ja/i18n/issues

To update you, @masamiy has got in contact with us from Australia and we had a conference call recently. She is able to train new instructors so we are discussing that as well. Sorry I wasn't able to contact you as your email bounced back. If you have an updated address we can use that. We also use the "swcarpentry" channel on the Tokyo.R [slack channel](https://r-wakalang.herokuapp.com/) quite a lot now.

**Comment by TomKellyGenetics** [comment]
@joelnitta If this review gets accepted, mergify should merge automatically, right?

**Comment by joelnitta** [comment]
@TomKellyGenetics yes, that's my understanding. I suppose that will happen when all of the conversations in the review have been resolved? It's a little confusing because it says "all checks have passed" even though the review is still not fully resolved.

**Comment by TomKellyGenetics** [comment]
@joelnitta We had a merge conflict on the headers that I've fixed. Maybe Mergify stalled on that so we have to do this one manually.

Thanks @atusy @masamiy

**Comment by TomKellyGenetics** [comment]
Sorry everyone, I realised that I merged while the reviews were still being discussed. I've set up a followup #87 with changes suggested  by @masamiy. This resolves some formatting problems but please check that it looks okay.

File: po/r-novice-gapminder.ja.po (hunk 8 above)
Comment (masamiy) [reply po/r-novice-gapminder.ja.po:15642]: An idea:
データをクリーンアップするスクリプトを書きましょう。

**Comment by masamiy** [comment]
I might have messed up. Where should I go now to have a look?

**Comment by TomKellyGenetics** [comment]
Sorry I had to re-open a new Pull Request. You can check the current version of the lesson here: https://github.com/swcarpentry-ja/i18n/pull/87/files

**Pull request merged**